### ISSUE TITLE
Add support to change field visibility at runtime in Manager

### ISSUE THREAD

**Issue by azizaparveen** [Esri/solutions-components#350]
Feature table already have a built in function for that. Wondering if we can add it to the overflow menu
https://developers.arcgis.com/javascript/latest/sample-code/widgets-featuretable/

![image](https://github.com/Esri/solutions-components/assets/138827632/83176f34-9321-4b8f-b0e1-767b09b83dad)

**Comment by avezina** [comment]
@azizaparveen Would we still have a config for this as well, and would this allow overriding the config?

**Comment by azizaparveen** [comment]
> @azizaparveen Would we still have a config for this as well, and would this allow overriding the config?

Yes, in beta we are still planning to  have it in the config and in the full release we can add this at the runtime and allow it to override the config settings

**Comment by azizaparveen** [comment]
Verified in devext

**Comment by avezina** [comment]
Edge case, but should we allow hiding all columns? 
![image](https://github.com/Esri/solutions-components/assets/50338557/10fe0695-45ee-4d05-ac4a-b4f61312790e)

**Comment by avezina** [comment]
Two more questions on expected behavior:

1. If I show/hide fields at runtime and switch between maps or layers, and then come back to this layer: should I expect my visible fields election to have been preserved (right now it is reset)?
![image](https://github.com/Esri/solutions-components/assets/50338557/e79fd9f8-8b41-4d86-ad24-00b16edb8dc5)

2. If I choose to hide fields at runtime, should this affect which fields are exported in the CSV?

**Comment by azizaparveen** [comment]
@avezina 
1. I think it will be nice to preserve the setting for the session, but not sure if this is needed, not sure how common the workflow is for going back and forth between layers
2. This would be a nice feature, EXB table does this. 
Could you please open separate issues for these to track.
